bigquery/api/test/ImportCommandTest.php

```
@@ -0,0 +1,347 @@
+<?php
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Samples\BigQuery\Tests;
+
+use Google\Cloud\Samples\BigQuery\ImportCommand;
+use Google\Cloud\Samples\BigQuery\QueryCommand;
+use Google\Cloud\Samples\BigQuery\SchemaCommand;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * Unit Tests for ImportCommand
+ */
+class ImportCommandTest extends \PHPUnit_Framework_TestCase
+{
+    protected static $hasCredentials;
+    protected static $gcsBucket;
+
+    public static function setUpBeforeClass()
+    {
+        $path = getenv('GOOGLE_APPLICATION_CREDENTIALS');
+        self::$hasCredentials = $path && file_exists($path) &&
+            filesize($path) > 0;
+        self::$gcsBucket = getenv('GOOGLE_BUCKET_NAME');
```

I would fail early with a nice message if `getenv('GOOGLE_BUCKET_NAME')` is false